syntax = "proto3";
package portrait;




import "capacity_admin_sdk/model/capacity_admin/portrait_config.proto";

/*
CreatePortrait请求
*/
message CreatePortraitRequest {
    /*
    画像名称
    */
    string name = 1 ;
    /*
    画像编码
    */
    string charset = 2 ;
    /*
    画像负责人
    */
    string user = 3 ;
    /*
    画像描述
    */
    string memo = 4 ;
    /*
    画像配置
    */
    capacity_admin.PortraitConfig config = 5 ;
}

/*
CreatePortrait返回
*/
message CreatePortraitResponse {
    /*
    画像ID
    */
    string instanceId = 1 ;
}

/*
CreatePortraitApi返回
*/
message CreatePortraitResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    portrait.CreatePortraitResponse data = 4 ;
}

